pages/vi/vi-github-and-repositories.md

```diff
@@ -21,7 +21,7 @@ This is just a summary of the steps that you will need to perform. Please, keep
 
 
 * [Clone Your GitHub Repository username.github.io](#Clone_Your_GitHub_Repository_username.github.io)
-* [Clone with HTTPS or Use SSH?](#Clone_with_HTTPS_or_Use_SSH?)
+* [Clone with HTTPS or Clone with SSH?](#Clone_with_HTTPS_or_Use_SSH?)
```

Comment:
We probably want `#Clone_with_HTTPS_or_Use_SSH?` updated as well.

Reply:
@xyb994 I have updated it. Please take a look.